go/dynamodb/README.md

```diff
@@ -13,35 +13,123 @@ in the AWS SDK for Go Developer Guide.
 
 ## Running the Code
 
+### CreateTable/CreateTable.go
+
+This example creates a DynamoDB table.
+
+`go run CreateTable.go -t TABLE`
+
+- _TABLE_ is the name of the table.
+
+The unit test accepts a similar value in _config.json_.
+
+The table has two attributes:
+
+- **Year** is an integer
+- **Title** is a string
+
+### CreateTableItem/CreateTableItem.go
+
+Creates a new item in a DynamoDB table.
+
+`go run CreateTableItem -d TABLE -y YEAR -t TITLE -r RATING`
+
+- _TABLE_ is the name of the table.
+- _YEAR_ is the year that the movie was released.
+- _TITLE_ is the title of the movie.
+- _RATING_ is the rating, from 0.0 to 10.0, of the movie.
+
+The unit test accepts similar values in _config.json_.
+
+### DeleteItem/DeleteItem.go
+
+This example deletes an item from a DynamoDB table.
+
+`go run DeleteTable.go -t TABLE -m MOVIE -y YEAR`
+
+- _TABLE_ is the name of the table containing the item to delete.
+- _MOVIE_ is the name of the movie item to delete.
+- _YEAR_ is when the movie was released.
+
+The unit test mocks the DynamoDB service and the `DeleteItem` function.
+
+### GetItem/GetItem.go
+
+This example retrieves an item from a DynamoDB table.
+
+`go run GetItem.go -t TABLE -n NAME -y YEAR`
+
+- _TABLE_ is the name of the table
```

Comment:
Add ending periods to the list items (the rest in this README have them)

Reply:
Fixed.